test/methods/clipping/difference.jl

```diff
@@ -0,0 +1,68 @@
+# converts a vector of vector of vector of tuples to vec of vec of vec of array
+"""
+convert_tuple_to_array(Vector{Vector{Vector{Tuple{::Real}}}})::Vector{Vector{Vector{Vector{::Real}}}}
+
+    This function converts a vector of vectors of vectors of tuples to a 
+    vector of vectors of vectors of vectors.
+"""
+function convert_tuple_to_array(VVVT)
```

Comment:
Are these the same in union, difference, and intersection? If so, let's create a `clipping_test_utils` file or something and put them in there so that we don't have the repeat code. If they are different and I just don't see the difference then disregard